amaznot/app/Http/Controllers/LoginController.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class LoginController extends Controller
+{
+    public function index()
+    {
+        return view('auth.login');
+    }
+
+    
+    public function store(Request $request)
+    {   
+        // Validate
+        $this->validate($request,
+        [
+            'name' => 'required',
+            'password' => 'required|confirmed',
+        ]);
+
+        // Sign in User
+        if(!auth()->attempt($request-> only('name', 'password')))
+        {
+            return back()->with('status', 'Invalid login details');
+        }
+
+        // Redirect User 
+        if(auth()->user()->where('name', 1)->first());
+        {
+            return redirect()->route('productlist'); 
+        }
```

Comment:
What is done by where('name', 1)? Is the 1 meant to be hardcoded? Also when redirecting to the product list page, an aisle must be selected. Is that done here?

Reply:
'admin'

Reply:
Wasn't sure wherewould the admin be be redirected to., I'll just put him to home instead